docs-en/chapter_array_and_linkedlist/index.md

```diff
@@ -4,6 +4,6 @@
 
 !!! abstract
 
-    The world of data structures is like a solid brick wall.
+    The world of data structures resembles a sturdy brick wall.
 
-    The bricks of an array are neatly arranged, each closely connected to the next. In contrast, the bricks of a linked list are scattered, with vines of connections freely weaving through the gaps between bricks.
+    In arrays, envision bricks snugly aligned, each resting seamlessly beside the next, creating a unified formation. Meanwhile, in linked lists, these bricks disperse freely, embraced by sinuous vines gracefully knitting connections between them.
```

Comment:
"sinuous vines" -> "vines"

I suggest considering the removal of 'sinuous' to simplify the description. The term 'vines' alone effectively conveys the intended imagery.